inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_gather_elements.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "vpu/ngraph/transformations/dynamic_to_static_shape_gather_elements.hpp"
+
+#include "vpu/ngraph/operations/dynamic_shape_resolver.hpp"
+#include "vpu/ngraph/utilities.hpp"
+#include <vpu/utils/error.hpp>
+
+#include "ngraph/ops.hpp"
+
+#include <memory>
+
+namespace vpu {
+
+void dynamicToStaticShapeGatherElements(std::shared_ptr<ngraph::Node> target) {
```

Comment:
I don't see spec for Gather Elements. Could you please share it?
Before that, could you please explain why DTS should look this way. Seems a bit too simple. No shape subgraph, no checks if something should be static

Reply:
Added the spec.
It looks so simple just because the output shape is equal to the second input shape which is `indices` input.